_i18n/ko/_posts/2018/2018-01-16-prettier-1.10-secure-contexts-tdd-in-framework.md

```diff
@@ -0,0 +1,267 @@
+---
+title: "2018-01-16 JS: Prettier 1.10, Secure Contexts, TDD in Framework"
+author: "azu"
+2018-01-16-prettier-1.10-secure-contexts-tdd-in-framework.md
```

Comment:
@UYEONG Can you remove this line?
Instead of it, Add `translator` field.

https://travis-ci.org/jser/jser.github.io/builds/330163240#L706

> 
> YAMLException: can not read a block mapping entry; a multiline key may not be an implicit key at line 4, column 7:
>     layout: post

Reply:
OMG. I'm sorry.